app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div>
+    <v-form>
+      <v-row>
+        <v-col cols="12" xl="4">
+          <v-text-field
+            dense
+            flat
+            label="Name"
+            data-test="text-formName"
+            v-model="formName"
+          />
+        </v-col>
+        <v-col cols="12" xl="8">
+          <v-text-field
+            dense
+            flat
+            label="Description"
+            data-test="text-formDescription"
+            v-model="formDescription"
+          />
+        </v-col>
+      </v-row>
+      <v-btn color="primary" @click="submitFormSchema">
+        <span>Save Form Design</span>
+      </v-btn>
+    </v-form>
+    <FormBuilder :form="formSchema" @change="onChangeMethod" :options="{}" />
+  </div>
+</template>
+
+<script>
+import { FormBuilder } from 'vue-formio';
+import formService from '@/services/formService';
+
+export default {
+  name: 'FormDesigner',
+  components: {
+    FormBuilder
+  },
+  data: () => ({
+    formName: '',
+    formDescription: '',
+    formSchema: {}
+  }),
+  created() {
+    this.getFormSchema();
+  },
+  methods: {
+    async getFormSchema() {
+      this.formSchema = {};
+      try {
+        if (this.formId) {
+          const form = await formService.readForm(this.formId);
+          this.formName = form.data.name;
+          this.formDescription = form.data.description;
+
+          if (this.formVersionId) {
+            const response = await formService.readVersion(this.formId, this.formVersionId);
+            this.formSchema = response.data.schema;
+          }
+        }
+      } catch (error) {
+        console.error(`Error loading form schema: ${error}`); // eslint-disable-line no-console
+      }
+    },
+    async submitFormSchema() {
```

Comment:
Just need to be very clear that everyone realizes at this time, the form must exist and have an existing version for this component to work.  This only updates the specified version.

Reply:
Yeah lets address Create New Form after

Reply:
This component will be able to handle both scenarios, but yea - right now only Update (not Create) has been implemented. Will need to figure out the create flow pretty soon.